tests/wpunit/SSP_Episode_Functions_Test.php

```
@@ -0,0 +1,336 @@
+
+		$this->assertIsArray( $podcasts );
+		$this->assertEmpty( $podcasts );
+	}
+
+	/**
+	 * @covers ssp_episode_sync_status()
+	 */
+	public function test_ssp_episode_sync_status() {
+		$this->assertTrue( function_exists( 'ssp_episode_sync_status' ) );
+
+		$sync_status = ssp_episode_sync_status( $this->test_episode_id );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$sync_status'. (Unused Code Rules)

(UnusedLocalVariable)